app/keepers/keepers.go

```diff
@@ -106,12 +106,12 @@ type AppKeepers struct {
 	AuthzKeeper         authzkeeper.Keeper
 
 	StrategicreserveKeeper strategicreservemodulekeeper.Keeper
```

Comment:
Can we make StrategicreserveKeeper --> StrategicReserveKeeper for consistency?

Reply:
Done